python/cuml/cuml/manifold/t_sne.pyx

```diff
@@ -276,7 +276,12 @@ class TSNE(UniversalBase,
     _hyperparam_interop_translator = {
         "n_components": {
             3 : "NotImplemented",
-        }
+        },
+        "method": {
+            "exact": "exact",
+            "barnes_hut": "NotImplemented",
```

Comment:
Most users will default to Barnes hut since it's the default, so this will be very impactful for users of zero code change, and there is no good reason why we shouldn't change to fft here

Reply:
Can you weigh in on [this discussion](https://github.com/rapidsai/cuml/pull/6619#discussion_r2075711856) then?